### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Justman10000**

### CODE REVIEW

**Comment by Justman10000** [comment]
@kerhub

**Comment by Justman10000** [comment]
@geromegrignon

File: src/app/features/repository-finder/repository-finder.component.html [hunk 1]
```diff
@@ -1,16 +1,16 @@
 <form [formGroup]="form">
-  <label for="repository-url">Github repository url</label>
+  <label for="repository-URL">Github repository URL</label>
```
Comment (geromegrignon) [inline src/app/features/repository-finder/repository-finder.component.html:2]: Please roll back this change as IDs in the project don't use uppercase.

File: src/app/features/repository-finder/repository-finder.component.html [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
   <input
-    id="repository-url"
+    id="repository-URL"
     class="input"
-    type="url"
-    placeholder="enter the url for an enhanced experience"
+    type="URL"
```
Comment (geromegrignon) [inline src/app/features/repository-finder/repository-finder.component.html:6]: Please roll back this change as types in the project don't use uppercase.

File: package.json [hunk 3]
```diff
@@ -31,6 +31,7 @@
     "@angular/platform-browser-dynamic": "~13.3.11",
     "@angular/router": "~13.3.11",
     "@briebug/jest-schematic": "^2.1.1",
+    "ajv-formats": "2.1.1",
```
Comment (geromegrignon) [inline package.json:34]: this dependency is never used

**Review by geromegrignon — CHANGES_REQUESTED** [review]
Thanks for your contribution to this project.
Besides added comments, please remove yarn.lock and add back package-lock.json file.

File: src/app/features/repository-finder/repository-finder.component.html [hunk 4]
```diff
@@ -1,16 +1,16 @@
 <form [formGroup]="form">
-  <label for="repository-url">Github repository url</label>
+  <label for="repository-URL">Github repository URL</label>
   <input
-    id="repository-url"
+    id="repository-URL"
```
Comment (geromegrignon) [inline src/app/features/repository-finder/repository-finder.component.html:4]: Please roll back this change as IDs in the project don't use uppercase.

**Comment by Justman10000** [comment]
> ...add back package-lock.json file.

Why? As already mentioned in the commit, the package-lock.json is superfluous, since the content always remains exactly the same, unless the package.json is modified!

File: package.json (hunk 3 above)
Comment (Justman10000) [reply package.json:34]: As Netlify said:

grafik

File: src/app/features/repository-finder/repository-finder.component.html (hunk 1 above)
Comment (Justman10000) [reply src/app/features/repository-finder/repository-finder.component.html]: See c28e7a8

File: src/app/features/repository-finder/repository-finder.component.html (hunk 4 above)
Comment (Justman10000) [reply src/app/features/repository-finder/repository-finder.component.html]: See c28e7a8

File: src/app/features/repository-finder/repository-finder.component.html (hunk 2 above)
Comment (Justman10000) [reply src/app/features/repository-finder/repository-finder.component.html]: See c28e7a8

**Comment by Justman10000** [comment]
> ...add back package-lock.json file.

Why? As already mentioned in the commit, the package-lock.json is superfluous, since the content always remains exactly the same, unless the package.json is modified! In addition, this is generated by NPM!

**Comment by Justman10000** [comment]
> ...add back package-lock.json file.

Why? As already mentioned in the commit, the package-lock.json is superfluous, since the content always remains exactly the same, unless the package.json is modified! In addition, this is generated by NPM!

**Comment by geromegrignon** [comment]
> > ...add back package-lock.json file.
> 
> Why? As already mentioned in the commit, the package-lock.json is superfluous, since the content always remains exactly the same, unless the package.json is modified! In addition, this is generated by NPM!

That's the point of it, everyone is working on the same version. By adding the yarn.lock, you are acting the same way though.

**Comment by geromegrignon** [comment]
@Justman10000 in case you are participating in Hacktoberfest, I labelled this so valuable PR as accepted :)

**Comment by Justman10000** [comment]
> @Justman10000 in case you are participating in Hacktoberfest, I labelled this so valuable PR as accepted :)

You mean this?

grafik

**Comment by geromegrignon** [comment]
> > @Justman10000 in case you are participating in Hacktoberfest, I labelled this so valuable PR as accepted :)
> 
> You mean this?
> 
> grafik

Yes, waiting means the PR is participating.

**Comment by Justman10000** [comment]
@geromegrignon

When will this PR be merged?

**Comment by geromegrignon** [comment]
Hi @Justman10000, I was about to work on it :)

**Pull request merged**